src/OSPSuite.FuncParserNative/src/StringHelper.cpp

```diff
@@ -6,14 +6,10 @@ namespace FuncParserNative
 
 std::string StringHelper::Capitalize (const std::string & pInString)
 {
-	char * NewStr = new char[pInString.size()+1];
-	const char *csource = pInString.c_str();
-    for(size_t i=0; i<pInString.size(); i++)
-		NewStr[i] = i==0 ? toupper(csource[i]) : tolower(csource[i]);
-	NewStr[pInString.size()] = '\0';
-	std::string newString = NewStr;
-	delete[] NewStr;
-	return newString;
+    std::string newString = pInString;
+    newString[0] = toupper(pInString[0]);
```

Comment:
The code is not equivalent. It used to lower case other letters
But it's a convoluted way of doing it indeed

Reply:
> It used to lower case other letters...
lol true. merged too fast :)